### DISCUSSION TITLE
R2024_12_12

### DISCUSSION THREAD

**Discussion by SaketADumbre** [acpica/acpica#985] [Announcements]
This is the last ACPICA release of 2024! Happy Holidays to everyone and if you have any feedback regarding anything ACPICA related or ACPI in general, please feel free to email me at saket.dumbre@intel.com

[changes.txt](https://github.com/user-attachments/files/18117935/changes.txt)
[acpitests-unix-20241212.tar.gz](https://github.com/user-attachments/files/18117942/acpitests-unix-20241212.tar.gz)
[acpitests-win-20241212.zip](https://github.com/user-attachments/files/18117944/acpitests-win-20241212.zip)
[iasl-win-20241212.zip](https://github.com/user-attachments/files/18117945/iasl-win-20241212.zip)
[acpica-unix2-20241212.tar.gz](https://github.com/user-attachments/files/18117946/acpica-unix2-20241212.tar.gz)
[acpica-unix-20241212.tar.gz](https://github.com/user-attachments/files/18117947/acpica-unix-20241212.tar.gz)
[acpica-win-20241212.zip](https://github.com/user-attachments/files/18117948/acpica-win-20241212.zip)

<hr /><em>This discussion was created from the release <a href='https://github.com/acpica/acpica/releases/tag/'>R2024_12_12</a>.</em>

**Comment by ElyesH** [comment]
Happy Holidays
Would you please update https://www.intel.com/content/www/us/en/developer/topic-technology/open/acpica/download.html ?

**Reply by SaketADumbre** [reply]
Hi @ElyesH  I am working towards it to ensure the website is updated by the end of this week, thanks a lot!
